external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSinkSuite.scala

```diff
@@ -332,7 +332,9 @@ class KafkaSinkSuite extends StreamTest with SharedSQLContext with KafkaTest {
     var ex: Exception = null
     try {
       ex = intercept[StreamingQueryException] {
-        writer = createKafkaWriter(input.toDF(), withTopic = Some(topic))()
+        writer = createKafkaWriter(input.toDF(),
```

Comment:
Can we set this inside `createKafkaWriter`? it seems like it's a generally good idea to not let anything block that long in tests. In fact KafkaContinuousSinkSuite sets it to 1000. We should standardize that too.

Reply:
@srowen Thank you. I thought about it initially, but went with a conservative fix.

> it seems like it's a generally good idea to not let anything block that long in tests
I agree.

> In fact KafkaContinuousSinkSuite sets it to 1000. We should standardize that too
Yeah.. wasn't sure on value to set. If we think 1 second is a good enough timeout setting, i can change.

Reply:
In retrospect 1000 may have been too low - that could cause flakiness if the test runner is slow or something and it legitimately takes a second or two for Kafka to respond. 10k definitely should work, and I'd be fine with 1k if we're really worried about test time here.

Reply:
Honestly I think it's worth trying. Many Kafka tests are slow, and this could speed up many of them. What happens if it's 1 second everywhere? or even just 10?

Reply:
@srowen I have used 5 sec timeout for now and i have moved it to createKafkaWriter per ur suggestion. Pl.. let me know if you are okay. Currently there is only one test that make uses of this timeout value. So keep it conservatively at 5 secs ?